### DISCUSSION TITLE
help request: how to deploy push apisix log to skywalking?

### DISCUSSION THREAD

**Discussion by michaelh1355** [apache/apisix#7808] [Q&A]
### Description

hi ,

I setup the skywalking(oap/ui/elasticsearch) and it 's works now. now I want to push apisix log to skywalking system. is there any details document to me for reference? It is appreciate  to give a setup example. 

thank you!

### Environment

- APISIX version (run `apisix version`):
- Operating system (run `uname -a`):
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`):
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`):
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`):

**Comment by moonming** [comment]
https://apisix.apache.org/blog/2021/12/07/apisix-integrate-skywalking-plugin/
https://apisix.apache.org/docs/apisix/next/plugins/skywalking-logger/
please take a look

**Comment by michaelh1355** [comment]
hi,
https://apisix.apache.org/docs/apisix/plugins/skywalking-logger/
curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
{
      "plugins": {
            "skywalking-logger": {
                "endpoint_addr": "http://127.0.0.1:12800"
            }
       },
      "upstream": {
           "type": "roundrobin",
           "nodes": {
               "127.0.0.1:1980": 1
           }
      },
      "uri": "/hello"
![image](https://user-images.githubusercontent.com/90734605/186639016-4f7e5cc4-7c2f-40b9-81ec-8b42431b5dde.png)

}'

in this guide. what is the port 1980? there is not 1980 port on the server.

**Comment by tzssangglass** [comment]
> in this guide. what is the port 1980? there is not 1980 port on the server.

1980 is the port for the upstream service, I think you should read some documentation first. ref: https://github.com/apache/apisix/tree/master/docs/en/latest
